complaints/models.py

```diff
@@ -1,7 +1,8 @@
+"""Module for Creating Complaints Model"""
 from django.db import models
 from django.contrib.auth import get_user_model
 
-User = get_user_model()
+USER = get_user_model()
```

Comment:
The earlier variable naming scheme was correct according to the Django conventions. What was wrong, is the use of globals.

Reply:
ok .I will fix that